src/shared/Core/CurlCookie.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using GitCredentialManager;
+
+namespace CurlCookie
```

Comment:
The namespace should be `GitCredentialManager`:

```suggestion
namespace GitCredentialManager
```